xbmc/platform/win10/CPUInfoWin10.cpp

```diff
@@ -117,3 +117,11 @@ int CCPUInfoWin10::GetUsedPercentage()
 
   return static_cast<int>(m_lastUsedPercentage);
 }
+
+bool CCPUInfoWin10::GetTemperature(CTemperature& temperature)
+{
+  temperature = CTemperature::CreateFromCelsius(-1);
+  temperature.SetValid(true);
```

Comment:
why set the temperature to valid?

Reply:
to allow it to pass the test

Reply:
Adding incorrect application code, to fix a wrong test case doesn't sound sane for me.

Reply:
if you want to do the windows implementation be my guest

Reply:
```
bool CCPUInfoWin10::GetTemperature(CTemperature& temperature)
{
  temperature.SetValid(false);
  return false;
}
```

is the correct implementation.